test/lint/streams.rb

```diff
@@ -90,6 +90,14 @@ def test_xadd_with_maxlen_and_approximate_option
       assert_match ENTRY_ID_FORMAT, actual
     end
 
+    def test_xadd_with_nomkstream_option
```

Comment:
You need a call to `omit_version` here so that this test doesn't run against redis 6 and older.

Reply:
Please squash your commits though.

Reply:
Done.